apps/myaccount/package.json

```diff
@@ -39,8 +39,8 @@
         "@babel/polyfill": "^7.0.0",
         "@mui/material": "^5.13.0",
         "@mui/system": "^5.12.3",
-        "@oxygen-ui/react": "^1.13.0",
-        "@oxygen-ui/react-icons": "^1.13.0",
+        "@oxygen-ui/react": "^1.13.1",
+        "@oxygen-ui/react-icons": "^1.13.1",
```

Comment:
Shall we move these packages to the root package.json as these are used by almost all components in the project?

@brionmario @JayaShakthi97 WDYT?

Reply:
Lets keep them in the package level itself. Since these features are meant to be self contained and bundled with `rollup` in the long run.

Reply:
Additionally, with package based nx, we need to scope the dependencies to the relevant package itself to aviod `extraneous-dependencies`.

Reply:
Ack. thanks.